pyinfra/api/connectors/mech.py

```diff
@@ -0,0 +1,162 @@
+import json
+
+from os import path
+from threading import Thread
+
+from six.moves.queue import Queue
+
+from pyinfra import local, logger
+from pyinfra.api.exceptions import InventoryError
+from pyinfra.api.util import memoize
+from pyinfra.progress import progress_spinner
+
+
+def _get_mech_ssh_config(queue, progress, target):
+    logger.debug('Loading SSH config for {0}'.format(target))
+
+    queue.put(local.shell(
```

Comment:
One possible "fix" for the host issue is to replace `Host mech` with `Host {target}` in the first line before putting the output onto the queue.

Reply:
Note this will work because we use the host value to identify the pyinfra host - ie all the hosts being `mech` will override each other (https://github.com/Fizzadar/pyinfra/pull/198/files#diff-89a12c5b857ea27a0a0716fcbf901317R118).

Reply:
Thanks. That was a great suggestion.